topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: tutorial_hands_on
+
+title: Screening assembled genomes for contamination using NCBI FCS
+zenodo_link: 'https://zenodo.org/records/10881551'
+questions:
+- Are the sequences in a genome assembly contaminant-free?
+objectives:
+- Learn how to screen a genome assembly for adaptor and vector contamination.
+- Learn how to screen a genome assembly for non-host organism contamination.
+time_estimation: 90M
+key_points:
+- Contamination can arise from multiple sources and occur at multiple stages of a genome assembly project.
+- Contamination cleanup is essential for reliable comparative genomics analyses.
+contributors:
+- tvedtees
+
+---
+
+
+# Introduction
+
+<!-- This is a comment. -->
```

Comment:
```suggestion
```

i know this was part of the template but we avoid the introduction section as the first paragraph gets pulled as an 'abstract'